packages/suite/src/views/onboarding/steps/SecurityCheck/SecurityCheck.tsx

```diff
@@ -231,7 +231,7 @@ export const SecurityCheck = () => {
                                             color={theme.TYPE_WHITE}
                                         />
                                     </IconWrapper>
-                                    <Translation id="TR_TAKES_N_MINUTES" values={{ n: '5' }} />
+                                    <Translation id="TR_TAKES_N_MINUTES" values={{ n: '20' }} />
```

Comment:
I see no point of having the constant defined here in Suite as value. The number can be part of translation string...

Reply:
Ok.. I've removed the values param, updated it in crowdin, and notified Chris to confirm the changes...